### PR TITLE
feat: add env attribute to webpack_bundle

### PR DESCRIPTION
**Pull request by gregmagolan**
Resolves https://github.com/aspect-build/rules_webpack/issues/63

### CODE REVIEW

File: webpack/private/webpack_bundle.bzl [hunk 1]
```diff
@@ -184,6 +190,12 @@ def _impl(ctx):
     else:
         args.add_all(["--output-path", output_sources[0].short_path[:-len(output_sources[0].basename)]])
 
+    env = {
+        "BAZEL_BINDIR": ctx.bin_dir.path,
```
Comment (kormide) [inline webpack/private/webpack_bundle.bzl:194]: Why hardcode this? Isn't it available as `$(BINDIR)`?

File: webpack/tests/simple/webpack.config.js [hunk 2]
```diff
@@ -1 +1,8 @@
+if (process.env.MY_ENV != 'webpack/tests/simple/index.js') {
```
Comment (kormide) [inline webpack/tests/simple/webpack.config.js:1]: I'm confused. Doesn't an execpath include the three output tree prefixes?

Comment (gregmagolan) [reply webpack/tests/simple/webpack.config.js:1]: This is a source file so it doesn't in this case

Comment (kormide) [reply webpack/tests/simple/webpack.config.js:1]: Ah, right.

File: webpack/private/webpack_bundle.bzl (hunk 1 above)
Comment (gregmagolan) [reply webpack/private/webpack_bundle.bzl:194]: `BAZEL_BINDIR` a mandatory env var that is needed whenever running a js_binary tool. Setting it to `ctx.bin_dir.path` is simpler than expanding a `$(BINDIR)` string to the same value.

**Review by kormide — APPROVED** [review]

**Pull request merged**